src/braces.cpp

```diff
@@ -858,7 +858,7 @@ void add_long_closebrace_comment(void)
                {
                   nl_min = cpd.settings[UO_mod_add_long_switch_closebrace_comment].u;
                   tag_pc = sw_pc;
-                  xstr   = sw_pc ? sw_pc->str : NULL;
+                  xstr   = sw_pc ? sw_pc->str : NULL; /* \todo NULL is no unc_text structure */
```

Comment:
It is, though I am not really sure if this results in the correct behavior here. Basically the [operator=](https://github.com/uncrustify/uncrustify/blob/8004bdac1d6d9b495668daa9b502c11f99879cdd/src/unc_text.h#L82) function of `unc_text`  is called. This sets the content of `xstr` to `0`.